changelogs/fragments/update_dependency.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- Update dependency versions in galaxy.yml.
```

Comment:
No changelog fragment needed here. (There has not been a proper release yet, the last proper release is Ansible 2.9 :) )